app/components/PickBakerStakeAmount.tsx

```diff
@@ -84,6 +96,14 @@ export default function PickBakerStakeAmount({
                 />
             </div>
             <ErrorMessage>{errors[fieldName]?.message}</ErrorMessage>
+            {cooldownUntil && showCooldown && (
+                <div className="textFaded">
+                    Will take effect at
```

Comment:
Because it makes more sense in the context of changing the value, whereas the other description was more fitting in the context of describing the grace period. If you have a better suggestion I'd love to hear it 🙂

Reply:
Maybe "Takes effect after"?

Reply:
I guess I'm just confused because you didn't change the context here, unless I'm misunderstanding.
On both the baker and delegation page, we still talk about a grace period in the text.

Reply:
I guess you moved its location in the component.
"Will take effect at" > "Takes effect after", maybe "would take effect at"? (What does the MW say?)